docs/proposals/sla-aware-warmpool-routing.md

```
@@ -0,0 +1,415 @@
+          - name: pool-runc
+      hardware:
+        pools:
+          - name: pool-kata
+```
+
+Pools are listed in preference order within each tier. The webhook tries them
+sequentially.
+
+### Webhook flow
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)